assembl/static2/js/app/reducers/adminReducer/index.js

```diff
@@ -63,10 +49,47 @@ export const thematicsById: ThematicsByIdReducer = (state = Map(), action) => {
       titleEntries: List(),
       video: null
     });
-    return state.set(action.id, emptyThematic.set('id', action.id));
+    const order = state.size + 1.0;
+    return state.set(action.id, emptyThematic.set('id', action.id).set('order', order));
   }
   case 'DELETE_THEMATIC':
     return state.setIn([action.id, '_toDelete'], true);
+  case 'MOVE_THEMATIC_UP': {
+    let newState = state;
```

Comment:
is there a way to put this stuff in a function and to use it for all reordable items ? (since all items have `id`, `order`, `_toDelete` and `_hasChanged` fields)